packages/ember/index.ts

```diff
@@ -1,24 +1,24 @@
-import { DEPRECATIONS, deprecateUntil } from '@ember/-internals/deprecations';
+// import { DEPRECATIONS, deprecateUntil } from '@ember/-internals/deprecations';
 import doNotUseThis from './barrel';
 
 export default new Proxy(doNotUseThis, {
   get(target, key, receiver) {
     // We don't have symbol exports, so this is probably fine.
     if (typeof key === 'string') {
-      deprecateUntil(
```

Comment:
why comment this out 😅

Reply:
@NullVoxPopuli seems deprecation generated on bundle stage

Reply:
hmm. :( 

it _should_ be during regular build (at the same time we strip TS and stuff)

Reply:
restored